### PR TITLE
Add Update Phase API

### PR DESCRIPTION
**Pull request by harry-kim**
Refs: #SDE-3635

TODO:
- [x] Set milestone label in pull request (SDE version)
- [x] Updated changelog.md
- [x] Completed documentation for the new endpoint.
- [x] Feature has been merged into SDE develop.

### CODE REVIEW

**Comment by aeldamaty** [comment]
+1

File: source/includes/phases.md [hunk 1]
```diff
@@ -83,3 +85,161 @@ This endpoint retrieves a specific Phase resource, as specified by the id parame
...
+**`POST /api/v2/phases/{phase_id}/`**
+
+Fields             | Required | Description
+-------------------|----------|-------------
+name               | Yes      | The name of the phase.
+description        | No       | The description of the phase.
+tip                | No       | The tooltip for the phase.
+ordinal            | No       | The order of the phase.
+active             | No       | Whether or not the phase is active.
+retain             | No       | Retain this phase when creating a new release.
```
Comment (JeffBain) [inline source/includes/phases.md]: What does it mean to retain the phase again? Is a little unclear. Does this mean like "Keep the task statuses the same" or something similar?

Comment (harry-kim) [inline source/includes/phases.md]: means keep this custom phase in the new release

Comment (funkaoshi) [inline source/includes/phases.md]: We copy over the phases statuses and notes.

File: source/includes/changelog.md [hunk 2]
```diff
@@ -1,7 +1,8 @@
 # Changelog
 
 ## v4.5
-- Add 'ordinal', 'active', and 'is_custom' fields to phases returned by endpoint
+
+- Add documentation for editing a Phase
```
Comment (funkaoshi) [inline source/includes/changelog.md]: I'd reword to: "Phases can be created, edited, or deleted."

File: source/includes/phases.md (hunk 1 above)
Comment (funkaoshi) [inline source/includes/phases.md]: Also, should say that these two fields just set whether this is on or not by default in the new release form.

File: source/includes/changelog.md (hunk 2 above)
Comment (harry-kim) [inline source/includes/changelog.md]: updated

File: source/includes/phases.md (hunk 1 above)
Comment (JeffBain) [inline source/includes/phases.md]: Yeah maybe more of that detail.

"Determines whether this phase carries over statuses and notes by default"

Comment (harry-kim) [inline source/includes/phases.md]: updated
